addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/HashDisclosureScanRule.java

```diff
@@ -58,10 +58,10 @@ public class HashDisclosureScanRule extends PluginPassiveScanner {
                 Pattern.compile("\\$K4\\$[a-f0-9]{16},", Pattern.CASE_INSENSITIVE),
                 new HashAlert("Kerberos AFS DES", Alert.RISK_HIGH, Alert.CONFIDENCE_HIGH));
         hashPatterns.put(
-                Pattern.compile("\\$2a\\$05\\$[a-zA-z0-9\\+\\-_./=]{53}", Pattern.CASE_INSENSITIVE),
+                Pattern.compile("\\$2a\\$05\\$[a-zA-Z0-9\\+\\-_./=]{53}", Pattern.CASE_INSENSITIVE),
```

Comment:
Can be removed instead, the pattern has case insensitive flag, same for the other case.

Reply:
OH damn, good catch.